client/js/lounge.js

```diff
@@ -129,6 +129,11 @@ $(function() {
 				feedback.hide();
 			});
 		}
+
+		if (data.token && window.localStorage.setItem("token") !== null) {
```

Comment:
I think you meant `getItem`, no? Also, shouldn't that token change only happen if `data.success`?